lib/Doctrine/DBAL/Driver/PDOSqlsrv/Driver.php

```diff
@@ -61,16 +66,58 @@ private function _constructPdoDsn(array $params)
         }
 
         if (isset($params['dbname'])) {
-            $dsn .= ';Database=' .  $params['dbname'];
+            $dsn .= ';Database=' . $params['dbname'];
         }
 
-        if (isset($params['MultipleActiveResultSets'])) {
-            $dsn .= '; MultipleActiveResultSets=' . ($params['MultipleActiveResultSets'] ? 'true' : 'false');
+        if (! empty($connectionOptions)) {
```

Comment:
This condition seems redundant. If the method is invoked with an empty set, it will return an empty string which is acceptable.